frontend/src/renderer/components/ImageImporter.svelte

```diff
@@ -0,0 +1,148 @@
+<script lang="ts">
+    import { FilePlusIcon } from "svelte-feather-icons";
+    import { processState, sendMessage, messageStore } from "@util/stores";
+    import { forEach, find } from "lodash";
+    import ImageBubble from "@components/Process/ImageBubble.svelte";
+    import Dropzone from "svelte-file-dropzone";
+    export let label = "Select Files";
+    export let icon = FilePlusIcon;
+    export let largeText = false;
+    let filePaths = [];
+    let files = {
+        accepted: [],
+        rejected: []
+    };
+    let MAX_FILES = 8
+
+    function getThumbnails() {
+        console.log("Getting Thumbnails");
+        $processState.thumbnailID = Math.floor(Math.random() * 999999999);
+        let msg = {
+            RequestID: $processState.thumbnailID,
+            RequestType: "Thumbnails",
+            RequestData: {
+                names: filePaths,
+            },
+        };
+        console.log(msg);
+        sendMessage(JSON.stringify(msg));
+        filePaths = [];
+    }
+
+    $: if (filePaths?.length > 0) {
+        console.log("Fetching Thumbnails");
+        getThumbnails();
+    }
+
+    $: if (
+        $processState.imageFilePaths.length >= 6 &&
+        !$processState.completedTabs[0]
+    ) {
+        $processState.completedTabs[0] = true;
+    }
+
+    function handleFilesSelect(e) {
+        console.log("files")
```

Comment:
Not a big deal, but I'd remove the console.logs just because they can just add to the clutter. On the other hand, some useful comments would be even better.